sdk/dapp-kit/src/hooks/wallet/useWallet.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useWalletContext } from '../../components/WalletProvider.js';
+import { useMemo } from 'react';
+
+/**
+ * Hook for retrieving wallet and account information.
+ */
+export function useWallet() {
```

Comment:
We talked about making this a bunch of hooks instead of one I thought

Reply:
Hmm, I actually think that this might be okay to have multiple hooks for each of these? I'm not 100% sure, and we can iterate on this in the future, but I think more often than not you're just grabbing the current account anyway, so having a bag of functionality that comes along for the ride isn't the most ideal.

Splitting hooks up _could_ improve performance if these each have their own provider, or if we implement some approach using selectors (maybe using zustand or jotai or whatever), but in the current approach yeah there's no perf benefit.

Reply:
But for now we can leave this as-is and iterate in the future

Reply:
Okay yeah, that makes sense and +1 to your point about the potential improvements to prevent re-renders. Let me get the base of all this fleshed out and we can iterate like you said!